etc/ci/update-wpt-checkout

```diff
@@ -115,7 +115,11 @@ EOF
     curl -H "Authorization: token ${WPT_SYNC_TOKEN}" \
          -H "Content-Type: application/json" \
          --data @prdata.json \
-         https://api.github.com/repos/servo/servo/pulls || return 5
+         https://api.github.com/repos/servo/servo/pulls |
+    jq '.review_comments_url' |
+    sed 's/pulls/issues/' |
+    xargs curl -H "Authorization: token ${WPT_SYNC_TOKEN}" \
+               --data "{\"body\":\"@bors-servo r+\"}" || return 5
```

Comment:
It will be easier to diagnose failure is the second curl command can return a different value than the first curl command if it fails.

Reply:
Thanks! It sounds great! Will update!

Reply:
@jdm Please help me check if the update is what you expect! Thanks! 🙇